### ISSUE TITLE
Add a WiFi status endpoint

### ISSUE THREAD

**Issue by larsblumberg** [getsenic/nuimo-hub-backend#28]
1. **`/-/setup/wifi/connection`**
tells which WiFi network it has joined:

```
{
  ssid: "GetOfMyLawn",
  status: "connected" | "connecting" | "invalid-password" | "unavailable"
}
```
2. **`/-/setup/wifi/adhoc`**
tells if it has opened a adhoc network:
```
{
  "name": "Senic Hub",
  "status": "available" | "unavailable"
}
```

**Comment by tomster** [comment]
implemented in #59

**Comment by tomster** [comment]
this has been implemented
